includes/Modules/AdSense.php

```diff
@@ -954,7 +952,7 @@ protected function validate_report_dimensions( $dimensions ) {
 					'Unsupported dimension requested: %s',
 					'google-site-kit'
 				),
-				$invalid_dimensions
+				$invalid_dimensions[0]
```

Comment:
Fixed the various AdSense/Analytics metric/dimension validation errors for single-value cases.
    
To date when provided with a single, invalid metric/dimension the returned message has been as follows, with "Array" instead of the metric/dimension name: _Unsupported (metric|dimension) requested: Array_.